src/cli/cli.cpp

```diff
@@ -553,20 +553,27 @@ void Interpreter::ProcessChannel(int argc, char *argv[])
             mServer->OutputFormat("enabled: %d\r\n", otChannelMonitorIsEnabled(mInstance));
             if (otChannelMonitorIsEnabled(mInstance))
             {
+                uint8_t  channel     = 0;
+                uint32_t channelMask = otLinkGetPhySupportedChannelMask(mInstance);
```

Comment:
As @abtink suggested in #3732, we will not expose otLinkGetPhySupportedChannelMask() as a public API, so it's removed by #3732.

Reply:
Got it. I will close this PR.